package.json

```diff
@@ -34,13 +34,8 @@
     "build": "yarn workspace @maticoapp/matico_charts build-prod && yarn workspace @maticoapp/matico_components build-prod &&  (cd matico_app_server && yarn run prisma generate && yarn build )",
     "deploy": "yarn predeploy && yarn run netlify deploy --prod",
     "build_rust_docs": "cargo doc --open --document-private-items --no-deps",
-    "lint": "gts lint",
-    "clean": "gts clean",
-    "compile": "tsc",
-    "fix": "gts fix",
-    "prepare": "yarn run compile",
-    "pretest": "yarn run compile",
-    "posttest": "yarn run lint"
+    "prettier-check": "yarn prettier --check '**/*.{js,ts,jsx,tsx}'",
+    "prettier-write": "yarn prettier --write '**/*.{js,ts,jsx,tsx}'"
```

Comment:
here's the issue that was creating the problem: https://github.com/yarnpkg/berry/issues/1792#issuecomment-686527222